### PR TITLE
Update machine.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the Possible values for onboarding status.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9711

File: microsoft-365/security/defender-endpoint/machine.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ search.appverid: met150
 |firstSeen|DateTimeOffset|First date and time where the [machine](machine.md) was observed by Microsoft Defender for Endpoint.|
 |lastSeen|DateTimeOffset|Time and date of the last received full device report. A device typically sends a full report every 24 hours.|
 |osPlatform|String|Operating system platform.|
-|onboardingstatus|String|Status of machine onboarding. Possible values are: "onboarded" and "offboarded".|
+|onboardingstatus|String|Status of machine onboarding. Possible values are: "onboarded", "CanBeOnboarded", "Unsupported" amd "InsufficientInfo" .|
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/machine.md]:
```suggestion
|onboardingstatus|String|Status of machine onboarding. Possible values are: "onboarded", "CanBeOnboarded", "Unsupported", and "InsufficientInfo".|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Pull request merged**